repos/testUtils/support/world.js

```diff
@@ -1,9 +1,18 @@
-const { setWorldConstructor, World } = require('@cucumber/cucumber')
+const { setWorldConstructor } = require('@cucumber/cucumber')
+const { getClientWorld } = require('./getClientWorld')
+const { deepMerge } = require('@keg-hub/jsutils')
 
-class HerkinWorld extends World {
-    constructor(options) {
-        super(options)
-    }
-}
+/**
+ * Merge of world defaults with client world
+ */
+const HerkinWorld = deepMerge({
+  app: {
+    url: process.env.HERKIN_APP_URL
+  }
+}, getClientWorld())
 
-setWorldConstructor(HerkinWorld)
\ No newline at end of file
+setWorldConstructor(function (options={}) {
```

Comment:
What is this doing exactly? Do we still need it.
It's importing form `cucumber`, which doesn't make sense to me?

Reply:
not needed anymore, removed